core/src/main/java/com/linecorp/armeria/server/VirtualHostBuilder.java

```diff
@@ -241,10 +245,55 @@ public VirtualHostBuilder tls(KeyManagerFactory keyManagerFactory,
         requireNonNull(keyManagerFactory, "keyManagerFactory");
         requireNonNull(tlsCustomizer, "tlsCustomizer");
 
-        tls(buildSslContext(() -> SslContextBuilder.forServer(keyManagerFactory), tlsCustomizer));
+        final SslContext sslContext = buildSslContext(() -> SslContextBuilder.forServer(keyManagerFactory),
+                                                      tlsCustomizer);
+        try {
+            validateKeyStoreNullPassword(sslContext);
+        } catch (Exception e) {
+            throw new RuntimeException("failed to validate SSL/TLS configuration: " + e.toString(), e);
+        }
+
+        tls(sslContext);
         return this;
     }
 
+    /**
+     * Validate key store which is used for building {@code sslContext}.
+     * If key store is initialized with null password, an exception will be raised.
+     */
+    private void validateKeyStoreNullPassword(SslContext sslContext) throws Exception {
```

Comment:
- How about merging this method into `validateSslContext()`? We also should remove the `validateSslContext()` call in the `VirtualHost`'s constructor to avoid double validation, because now we validate the `SslContext` early in `tls(SslContext)`.
- Is it possible to make this method throw an `SSLException` rather than an `Exception`? If so, we will not have to catch and wrap it because `tls()` can throw an `SslException` already.

Reply:
* I merged `validateKeyStoreNullPassword()` into `validateSslContext()` 
and removed the `validateSslContext()` call in the `VirtualHost`'s constructor.
* I made modified `validateSslContext()`, `ServerBuilder.tls(SslContext)`, `VirtualHostBuilder.tls(SslContext)` throws `SSLExcpetion`